### PR TITLE
Update information-barriers-multi-segment.md

### LINKED ISSUE #11597: Set-PolicyConfig -InformationBarrierMode 'MultiSegment' - Unrecognized mode
I have a completely new demo tenant, Information barriers are not configured.
InformationBarrierMode is SingleSegment.

I am running this command to enable multi-segment mode:

`Set-PolicyConfig -InformationBarrierMode 'MultiSegment'`

But I receive this error:

`
Information Barrier config is not updated successfully. Error: {"error":{"code":"40005","message":"Unrecognized mode, source mode and 
destination mode should be 'Single' or 'MultiAllow'"}}.
    + CategoryInfo          : NotSpecified: (:) [Set-PolicyConfig], IBConfigUpdateFailureException
    + FullyQualifiedErrorId : [Server=MW2NAM10WS013,RequestId=4ec525ad-3294-4163-ae0c-c603cd2fbb68,TimeStamp=06.04.2023 08:50:11] [Failu 
   reCategory=Cmdlet-IBConfigUpdateFailureException] DF42E1F2,Microsoft.Office.CompliancePolicy.Tasks.SetPolicyConfig
    + PSComputerName        : nam10b.ps.compliance.protection.outlook.com
`

I already installed the preview 2 of ExchangeOnlineManagement  3.2.0.

Are there additional steps necessary, which are not documented yet?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: de406555-1a6d-896b-ff10-f873f96c36aa
* Version Independent ID: de406555-1a6d-896b-ff10-f873f96c36aa
* Content: [Use multi-segment support in information barriers - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/information-barriers-multi-segment?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/information-barriers-multi-segment.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/information-barriers-multi-segment.md)
* Service: **o365-seccomp**
* GitHub Login: @robmazz
* Microsoft Alias: **robmazz**

### LINKED ISSUE #11773: Set-PolicyConfig -InformationBarrierMode 'MultiSegment' Does Not Work
Hello,

 I have attempted this with multiple tenants it does not work. I have tried via powershell 5 or terminal powershell 7. I have tried latest EXO 2.x and 3.x.
 
 This is a Microsoft issue not an issue to redirect to customer support it's obviously reoccurring in various licenses & tenant types as well as for various others. 

This contradicts documentation provided by Microsoft making the product look as if it'll work for a scenario when it doesn't.

> 
>  Set-PolicyConfig -InformationBarrierMode 'MultiSegment'
> Information Barrier config is not updated successfully. Error: {"error":{"code":"40005","message":"Unrecognized mode, source mode
> and destination mode should be 'Single' or 'MultiAllow'"}}.
>     + CategoryInfo          : NotSpecified: (:) [Set-PolicyConfig], IBConfigUpdateFailureException
>     + FullyQualifiedErrorId : [Server=MW2NAM12WS020,RequestId=365ec443-3d37-42ff-b60f-0d9c9f56bf2f,TimeStamp=4/19/2023 10:47:35 PM]
>     [FailureCategory=Cmdlet-IBConfigUpdateFailureException] DD30B15B,Microsoft.Office.CompliancePolicy.Tasks.SetPolicyConfig
>     + PSComputerName        : nam12b.ps.compliance.protection.outlook.com

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the PS Cmdlet to check the config

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11597

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11773

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/information-barriers-multi-segment.md [hunk 1]
```diff
@@ -89,7 +89,9 @@ With segments and policies defined, the North School District applies the polici
 If you want to support assigning users to multiple segments, you'll need to verify that your IB organization supports multiple segments. Run the following cmdlet to verify your IB mode:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-multi-segment.md]:
```suggestion
If you want to support assigning users to multiple segments, you'll need to verify that your IB organization supports multiple segments. Run the following cmdlets to verify your IB mode:
```

File: microsoft-365/compliance/information-barriers-multi-segment.md [hunk 2]
```diff
@@ -89,7 +89,9 @@ With segments and policies defined, the North School District applies the polici
...
 
 ```powershell
-Get-PolicyConfig
+Connect-IPPSSession
+connect-exchangeonline
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-multi-segment.md]:
```suggestion
Connect-ExchangeOnline
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
Closing as per the comment in #11597

**Pull request closed without merging**